### PR TITLE
revert to dvc repro (instead of dvc exp)

### PR DESCRIPTION
**Pull request by gabrieltorresgamez**
dvc exp causes plots to not update in studio, trying dvc repro again

### CODE REVIEW

**Comment by ghost** [comment]
# RUN: 2023-04-21 18-09-01

## Hyperparameters

| Path   | Param   | HEAD~1   | workspace   |
|--------|---------|----------|-------------|

## Metrics

| Path   | Metric   | HEAD~1   | workspace   | Change   |
|--------|----------|----------|-------------|----------|

# Plots

TODO

**Comment by ghost** [comment]
# RUN: 2023-04-21 18-14-01

## Hyperparameters

| Path        | Param                               | HEAD~1   | workspace   |
|-------------|-------------------------------------|----------|-------------|
| params.yaml | dvclive.data.batch_size             | 32       | 64          |
| params.yaml | dvclive.data.num_workers            | 0        | 12          |
| params.yaml | dvclive.model_hparams.learning_rate | 0.1      | 0.03        |
| params.yaml | dvclive.model_hparams.num_epochs    | 2        | 50          |
| params.yaml | dvclive.model_hparams.weight_decay  | 0.0005   | 0.001       |

## Metrics

| Path                 | Metric           | HEAD~1   | workspace   | Change   |
|----------------------|------------------|----------|-------------|----------|
| dvclive/metrics.json | epoch            | 2        | 50          | 48       |
| dvclive/metrics.json | step             | 354      | 4450        | 4096     |
| dvclive/metrics.json | test.epoch.acc   | 0.36129  | 0.38925     | 0.02796  |
| dvclive/metrics.json | test.epoch.f1    | 0.26923  | 0.30368     | 0.03444  |
| dvclive/metrics.json | test.step.acc    | 0.36129  | 0.38925     | 0.02796  |
| dvclive/metrics.json | test.step.f1     | 0.26923  | 0.30368     | 0.03444  |
| dvclive/metrics.json | train.epoch.acc  | 0.82433  | 0.98958     | 0.16525  |
| dvclive/metrics.json | train.epoch.f1   | 0.76904  | 0.98603     | 0.21699  |
| dvclive/metrics.json | train.epoch.loss | 0.50873  | 0.03544     | -0.47329 |
| dvclive/metrics.json | train.step.acc   | 0.96875  | 1.0         | 0.03125  |
| dvclive/metrics.json | train.step.f1    | 0.95789  | 1.0         | 0.04211  |
| dvclive/metrics.json | train.step.loss  | 0.15193  | 0.00216     | -0.14977 |
| dvclive/metrics.json | val.epoch.acc    | 0.8311   | 0.92721     | 0.09611  |
| dvclive/metrics.json | val.epoch.f1     | 0.7995   | 0.92157     | 0.12207  |
| dvclive/metrics.json | val.epoch.loss   | 0.62941  | 0.32745     | -0.30196 |
| dvclive/metrics.json | val.step.acc     | 0.71429  | 0.85714     | 0.14286  |
| dvclive/metrics.json | val.step.f1      | 0.5      | 0.66667     | 0.16667  |
| dvclive/metrics.json | val.step.loss    | 0.98827  | 0.17628     | -0.81199 |

# Plots

TODO

**Comment by gabrieltorresgamez** [comment]
shit's whack
